build.cake

```diff
@@ -206,28 +234,40 @@ Task("TestDriver")
   .WithCriteria(IsRunningOnWindows)
   .Does(() => 
 	{ 
-		RunTest(NUNIT3_CONSOLE, BIN_DIR, PORTABLE_AGENT_TESTS);
+		RunTest(NUNIT3_CONSOLE, BIN_DIR, PORTABLE_AGENT_TESTS, ref ErrorDetail);
 	});
 
 Task("TestAddins")
+    .OnError(exception =>
+        {
+            ErrorDetail.Add(exception.Message);
+        })
   .IsDependentOn("Build")
   .Does(() => 
 	{
-		RunTest(NUNIT3_CONSOLE, BIN_DIR, ADDIN_TESTS); 
+		RunTest(NUNIT3_CONSOLE, BIN_DIR, ADDIN_TESTS,"TestAddins", ref ErrorDetail); 
 	});
 
 Task("TestV2Driver")
   .IsDependentOn("Build")
+    .OnError(exception =>
+        {
+            ErrorDetail.Add(exception.Message);
+        })
   .Does(() => 
 	{ 
-		RunTest(NUNIT3_CONSOLE, BIN_DIR, V2_PORTABLE_AGENT_TESTS);
+		RunTest(NUNIT3_CONSOLE, BIN_DIR, V2_PORTABLE_AGENT_TESTS,"TestV2Driver", ref ErrorDetail);
 	});
 
 Task("TestConsole")
   .IsDependentOn("Build")
+    .OnError(exception =>
+        {
+            ErrorDetail.Add(exception.Message);
+        })
```

Comment:
Here and other places, I would like to see consistent indenting of the methods and the brackets.